kolibri/plugins/management/assets/src/views/manage-content-page/channels-grid.vue

```diff
@@ -65,6 +65,13 @@
     >
       {{ $tr('deleteFailureNotification') }}
     </ui-alert>
+    <ui-alert
```

Comment:
Yes they are both successes, but one is a more positive one (importing a channel) whereas the the other one (deleting a channel) is not as positive? Sounds good, I think I'm just over thinking it.